<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SupriyaSP" failures="2" tests="7" name="Default test(failed)" time="64.314" errors="0" timestamp="2024-08-06T13:03:43 IST">
  <testcase classname="practice.test.InvoiceTest" name="activateSim" time="0.048">
    <skipped/>
  </testcase> <!-- activateSim -->
  <testcase classname="practice.test.InvoiceTest" name="activateSim" time="0.004">
    <skipped/>
  </testcase> <!-- activateSim -->
  <testcase classname="practice.test.InvoiceTest" name="activateSim" time="0.003">
    <skipped/>
  </testcase> <!-- activateSim -->
  <testcase classname="practice.test.InvoiceTest" name="activateSim" time="0.006">
    <skipped/>
  </testcase> <!-- activateSim -->
  <testcase classname="practice.test.InvoiceTest" name="activateSim" time="0.004">
    <skipped/>
  </testcase> <!-- activateSim -->
  <testcase classname="practice.test.InvoiceTest" name="activateSim" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [login] but found []">
      <![CDATA[java.lang.AssertionError: expected [login] but found []
at practice.test.InvoiceTest.activateSim(InvoiceTest.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- activateSim -->
  <testcase classname="practice.test.InvoiceTest" name="createInvoiceTest" time="0.029">
    <failure type="java.lang.AssertionError" message="expected [Login] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]">
      <![CDATA[java.lang.AssertionError: expected [Login] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]
at practice.test.InvoiceTest.createInvoiceTest(InvoiceTest.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- createInvoiceTest -->
</testsuite> <!-- Default test(failed) -->
